docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -228,6 +228,13 @@ children:
       url: Integrating-with-api-connect.html
       output: 'web, pdf'
 
+    - title: 'Using MessagePack'
+      output: 'web, pdf'
+      children:
+      - title: 'How to use MessagePack with LoopBack'
```

Comment:
Are you expecting to have multiple pages under `Using MessagePack` section? To me, this feels like an extra nesting.

Reply:
Good point, I've removed the nesting.

It was taken from the [TypeORM extension](https://github.com/strongloop/loopback-next/blob/44145563407e0252fcd424841a88ac74b55f8ed8/docs/site/sidebars/lb4_sidebar.yml#L304-L309)

Reply:
Good point, I've removed the nesting.

It was initially taken from the [TypeORM extension](https://github.com/strongloop/loopback-next/blob/44145563407e0252fcd424841a88ac74b55f8ed8/docs/site/sidebars/lb4_sidebar.yml#L304-L309)